shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DMLStatement.g4

```diff
@@ -48,11 +48,45 @@ conditionalInsertElsePart
     ;
 
 insertIntoClause
-    : INTO tableName (AS? alias)?
+    : INTO dmlTableExprClause alias? columnNames?
     ;
 
 insertValuesClause
-    : columnNames? VALUES assignmentValues (COMMA_ assignmentValues)*
+    : VALUES assignmentValues
+    ;
+
+returningClause
+    : (RETURN | RETURNING) exprs INTO dataItem (COMMA_ dataItem)*
+    ;
+
+dmlTableExprClause
+    : dmlTableClause | dmlSubqueryClause | tableCollectionExpr
+    ;
+
+dmlTableClause
+    : tableName (partitionExtClause | AT_ dbLink)?
+    | (viewName | materializedViewName) (AT_ dbLink)?
+    ;
+
+partitionExtClause
+    : PARTITION (LP_ partitionName RP_ | FOR LP_ partitionKeyValue (COMMA_ partitionKeyValue) RP_)
+    | SUBPARTITION (LP_ subpartitionName RP_ | FOR LP_ subpartitionKeyValue (COMMA_ subpartitionKeyValue) RP_)
+    ;
+
+dmlSubqueryClause
+    : LP_ selectSubquery subqueryRestrictionClause? RP_
+    ;
+
+subqueryRestrictionClause
+    : WITH (READ ONLY | CHECK OPTION) (CONSTRAINT constraintName)?
+    ;
+
+tableCollectionExpr
+    : TABLE LP_ collectionExpr RP_ (LP_ PLUS_ RP_)?
```

Comment:
Where is the definition of collection expression in ORALCE site?

Reply:
Here's the definition:
![tableCollectionExpression](https://user-images.githubusercontent.com/48581379/123531344-7a427380-d721-11eb-8426-2c1249453163.png)